sound/soc/sof/core.c

```diff
@@ -306,7 +355,6 @@ static int sof_remove(struct platform_device *pdev)
 {
 	struct snd_sof_dev *sdev = dev_get_drvdata(&pdev->dev);
 
-	snd_soc_unregister_platform(&pdev->dev);
```

Comment:
is this going to work for topic/sof-v4.14?

Reply:
gah, part of the post 4.18+ component API updates. Will revert for 4.14.